include/odp/api/spec/traffic_mngr.h

```diff
@@ -1062,6 +1062,7 @@ typedef struct {
 	/** The peak information rate for this shaper profile.  The units for
 	 * this integer is in bits per second when packet_mode is
 	 * not TRUE while in packets per second when packet mode is TRUE.
+	 * This field is ignored when dual_rate is FALSE.
```

Comment:
This patch looks good but I would like to improve the commit message. My suggestion:

```
api: tm: ignore peak rate and peak burst when dual rate is false

Clarify that peak rate and peak burst is ignored when dual rate
is set to false, indicating that the single rate is commit rate.
```